src/menu.scss

```diff
@@ -5,351 +5,306 @@
+    height: 0.0625rem;
+    background-color: var(--sapGroup_ContentBorderColor);
   }
 
-  &--mobile {      // for mobile mode
+  &__item {
     @include fd-reset();
 
-    max-width: none;
+    position: relative;
+    border-radius: 0;
+    background-color: var(--sapList_Background);
 
-    .#{$block}__link {
-      padding: 0.75rem 1rem 0.75rem 1rem; // line-height=20px so 2.75rem or 44px-20px=24px/2= 12px or 0.75rem each
-      height: 2.75rem;
-      width: 100%;
-      max-width: none;
-
-      .#{$block}__title {
-        max-width: 100%;
-      }
-    }
-
-    .#{$block}__addon-before {
-      min-width: 2.75rem; // to account for item right padding of 16px or 1rem so 2.75rem-1rem = 1.75rem
-      width: 2.75rem; // to account for item right padding of 16px or 1rem so 2.75rem-1rem = 1.75rem
-      height: 1.25rem; // to account for item top padding of 12px or 0.75rem so 2.75rem-0.75rem-0.75rem = 1.25rem
-      margin: 0 0 0 -1rem;
-    }
-
-    .#{$block}__addon-after {
-      min-width: 2.75rem; // to account for item right padding of 16px or 1rem so 2.75rem-1rem = 1.75rem
-      width: 2.75rem; // to account for item right padding of 16px or 1rem so 2.75rem-1rem = 1.75rem
-      height: 1.25rem; // to account for item top padding of 12px or 0.75rem so 2.75rem-0.75rem-0.75rem = 1.25rem
-      margin: 0 -1rem 0 0;
-    }
-
-    @include fd-rtl() {
-      .#{$block}__addon-before {
-        margin: 0 -1rem 0 0; // since before and after icons switch places, their margins also switch
-      }
-
-      .#{$block}__addon-after {
-        margin: 0 0 0 -1rem; // since before and after icons switch places, their margins also switch
-      }
-    }
-  }
-
-  @mixin selected-menu() {
-    .#{$block}__title {
-      color: $fd-menu-item-color;
-    }
-
-    background-color: $fd-menu-item-background-color--selected;
-    border-bottom: var(--sapList_BorderWidth) solid var(--sapList_SelectionBorderColor);
-
-    .#{$block}__addon-before,
-    .#{$block}__addon-after {
-      color: $fd-menu-primary-icon-color;
+    &:first-child {
+      border-top-right-radius: $fd-menu-border-radius;
+      border-top-left-radius: $fd-menu-border-radius;
     }
 
-    @include fd-hover() {
-      background-color: $fd-menu-item-background-color--selected-hover;
+    &:last-child {
+      border-bottom-right-radius: $fd-menu-border-radius;
+      border-bottom-left-radius: $fd-menu-border-radius;
     }
   }
 
-  &__separator {
-    @include fd-reset();
-
-    height: 0.0625rem;
-    width: 100%;
-    background-color: $fd-menu-separator-color;
-  }
-
-  &__item {
-    @include fd-reset();
-    @include action-cursor();
-
-    border-radius: inherit;
-    background-color: $fd-menu-item-background-color;
-    position: relative;   // so that sub-menus align relatively to parent menu item
-  }
-
   &__link {
     @include fd-reset();
+    @include action-cursor();
+    @include fd-fiori-focus();
 
-    border-radius: inherit;
+    text-decoration: none;
```

Comment:
Maybe we should consider having someting like `fd-reset-link()`, Our docs styles often change colors/underline on hover with some weird transition time.

Reply:
Good idea, but maybe we should move this to `fd-reset()`?

Reply:
I'm not sure, I wouldn't add anything to `fd-reset`, it will increase size of our lib. Let's keep it as it is right now. Things like that should be included in separated PR.